pkgs/applications/networking/cluster/kubespy/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, buildGoModule, fetchFromGitHub, installShellFiles }:
+
+buildGoModule rec {
+  pname = "kubespy";
+  version = "0.5.1";
+
+  goPackagePath = "github.com/pulumi/kubespy";
```

Comment:
i believe this is unused with buildGoModule
```suggestion
```

Reply:
Thanks! Addressed and force-pushed.